docs/user-guide/glossary.md

```diff
@@ -71,6 +73,8 @@ The category [components](#component) of a [microgrid](#microgrid) belong to.
 Examples of category types are meters, inverters, batteries and sensors.
 
 
+### Assets
```

Comment:
Maybe add a high level description for assets and devices?  Like assets are the core, like generators or storage, that are meaningful in business context, vs devices that are infrastructure required for the operation of the assets?